.github/pull_request_template.md

```
@@ -1,90 +1,14 @@
-# Pull Request
+## Summary
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)